### ISSUE TITLE
Update connection example in README

### ISSUE THREAD

**Issue by strogiyotec** [r2dbc/r2dbc-postgresql#105]
According to README , in order to create insert statement I need to create connection from **PostgresqlConnectionFactory** and then call **createStatement** method

```
ConnectionFactory connectionFactory = new PostgresqlConnectionFactory(PostgresqlConnectionConfiguration.builder()
    .host("...")
    .port("...").  // optional, defaults to 5432
    .username("...")
    .password("...")
    .database("...")  // optional
    .build());

Mono<Connection> connection = connectionFactory.create();
```
And then
```
connection
    .createStatement("INSERT INTO person (id, first_name, last_name) VALUES ($1, $2, $3)")
    .bind("$1", 1)
    .bind("$2", "Walter")
    .bind("$3", "White")
    .execute()
```
But we don't have **createStatement** method because connection is instance of Mono , instead of PostgresConnection

**Comment by nebhale** [comment]
Yep, quite unclear.  The latter `connection` is intended to be the resolved instance within a `flatMap()` (because we'd [never assign](https://github.com/r2dbc/r2dbc-spi/blob/master/r2dbc-spi-test/src/main/java/io/r2dbc/spi/test/Example.java#L335-L336) that `Mono` to anything directly) but there's no way you'd ever know that.  Thanks for the catch and PR.

**Comment by mp911de** [comment]
Fixed via #106.
